src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -170,6 +184,18 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
             Instant.now());
 
     messageStore.addMessage(message);
+    UUID mentionedUser = notificationStore.getuserMentioned(message.getContent());
+    //Add notification if a user is mentioned
+      if(mentionedUser != null) {
+        String notificationinfo = String.format("%s mentioned you in %s: %s", username, conversationTitle, message.getContent());
+        Notification notification =
+                new Notification(UUID.randomUUID(),
+                                 user.getId(),
+                                 conversation.getId(),
+                                 notificationinfo,
+                                 mentionedUser);
+        notificationStore.addNotification(notification);
+      }
 
```

Comment:
Once a user creates/sends a message to a conversation, if a user is mentioned a notification is created with the user id of the author of mention, conversation, what was said in the mention and the id of the user that was mentioned